content/courses/native-onchain-development/serialize-instruction-data-frontend.md

```diff
@@ -416,8 +435,15 @@ import { FC } from 'react'
 import { Movie } from '../models/Movie'
 import { useState } from 'react'
 import { Box, Button, FormControl, FormLabel, Input, NumberDecrementStepper, NumberIncrementStepper, NumberInput, NumberInputField, NumberInputStepper, Textarea } from '@chakra-ui/react'
```

Comment:
I think these imports (and other code) may also need to be updated to reflect the new `movie-review-dapp` you made with `create-solana-dapp`

Reply:
fixed!